spec/models/institution_builder_spec.rb

```diff
@@ -552,20 +552,20 @@
           end
 
           it 'is set from Section702School' do
-            create :sec702_school, :institution_builder
+            sec702_school = create :sec702_school, :institution_builder
             described_class.run(user)
 
-            expect(institutions.first.sec_702).not_to be_nil
-            expect(institutions.first.sec_702).to be_falsey
+            expect(institutions.find_by(facility_code: sec702_school.facility_code).sec_702).not_to be_nil
+            expect(institutions.find_by(facility_code: sec702_school.facility_code).sec_702).to be_falsey
           end
 
           it 'prefers Sec702School over Section702' do
             create :weam, :institution_builder, :private
-            create :sec702_school, :institution_builder, sec_702: true
+            sec702_school = create :sec702_school, :institution_builder, sec_702: true
             create :sec702, :institution_builder
             described_class.run(user)
 
-            expect(institutions.first.sec_702).to be_truthy
+            expect(institutions.find_by(facility_code: sec702_school.facility_code).sec_702).to be_truthy
```

Comment:
I don't see how these spec changes are related to `webpacker` & `react`.

Reply:
Most of these tests were using https://apidock.com/rails/v5.2.3/ActiveRecord/FinderMethods/first instead of https://apidock.com/rails/v5.2.3/ActiveRecord/FinderMethods/find_by

which is an issue when you can't guarantee a clean test database for each run, additionally the assumption by a unit test shouldn't be made that the record is the only record in the database.  There have been other tests in past week or so that failed for same reason

Reply:
OK. Sounds like it's not at all related to `webpacker` and/or `react-rails`. Should probably move it into a separate PR.

Reply:
https://github.com/department-of-veterans-affairs/gibct-data-service/pull/491

Reply:
#491 is merged into master